servers/rendering/rasterizer_rd/rasterizer_canvas_rd.cpp

```diff
@@ -1280,7 +1308,7 @@ void RasterizerCanvasRD::canvas_render_items(RID p_to_render_target, Item *p_ite
 				state.light_uniforms[index].shadow_y_ofs = 0;
 			}
 
```

Comment:
```suggestion
state.light_uniforms[index].flags &= ~(3 << LIGHT_FLAGS_BLEND_SHIFT);
```

Same change as above. Need to reset the light flags